app/tests/test_authenticate.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import User
```

Comment:
Use the same import name as you used in the base.py file. Makes it easier for people to understand.